### PR TITLE
[NPS-574] Add js and scss files for NbbReturnForms

### PR DESCRIPTION
**Pull request by grovas**

### CODE REVIEW

File: src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const manufacturerSupportFormElementsNumber = 3;
+const productDetailsFormElementsNumber = 5;
+const manufacturerSupportForm = 'manufacturer_support_form';
+const productDetailsForm = 'product_details_form';
+const others = 'others';
```
Comment (djedrzejewski) [inline src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js]: Maybe call this variable like "otherElementsKey" or something like that? So when comparising, you will be sure, what you're comparing to.

Comment (grovas) [reply src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js]: My intention was use the same name as label in the template, as the value passed from select form. Going further by your suggestion I would like to change name to `otherProducts`.

File: src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const manufacturerSupportFormElementsNumber = 3;
+const productDetailsFormElementsNumber = 5;
+const manufacturerSupportForm = 'manufacturer_support_form';
```
Comment (bystro) [inline src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js]: Consider to add suffix `ClassName` to where we have to do with consts that hold class names - `manufacturerSupportFormClassName`

File: src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+const manufacturerSupportFormElementsNumber = 3;
```
Comment (bystro) [inline src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js]: I think we have to do more with `count` than with `number` - const name.

File: src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            : null;
+    } else {
+        elementsVisibility(manufacturerSupportFormClassName, true, manufacturerSupportFormElementsCount);
+        document.getElementsByClassName(productDetailsFormClassName).length > 0
+            ? elementsVisibility(productDetailsFormClassName, false, productDetailsFormElementsCount)
+            : null;
+    }
+}
+
+function elementsVisibility(divClass, visible, elementsNumber) {
```
Comment (bystro) [inline src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js]: `elementsNumber` -> `elementsCount`

**Review by bystro — APPROVED** [review]

**Review by djedrzejewski — APPROVED** [review]

**Comment by patpilus** [comment]
Please update CHANGELOG and plugin version in `composer.json`.

**Comment by patpilus** [comment]
Please change PR name to depict changes more precisely :)

**Pull request merged**